.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### What this PR does
+
+Please include a summary of the change and which issue is fixed or a link to the feature it implements.
+
+### Why is this needed?
+
+Please include relevant motivation and context of why this PR is necessary, sentry/linear/notion/...
+
+Solves:  
+STO-XXX  
+EMU-XXX
```

Comment:
Because this is an open-source project, I think we can skip this part.
What do you think?

Reply:
I don't actually think having it in there is really detrimental as most (if not all) PRs are from Feather people. This means in most cases it would be useful and might only be unnecessary in a minuscule amount of scenarios. And even then, people can just delete those lines, same as they'd do with all the other placeholder / description text in the template.

Maybe we could add "OPTIONAL" or something similar instead just to make it clear that it's not mandatory to have this in the PR?

Reply:
Yeah, you are right. Let's keep it as it is 😄